paymentsheet-example/src/androidTestDebug/java/com/stripe/android/test/core/FieldPopulator.kt

```diff
@@ -87,7 +88,7 @@ class FieldPopulator(
         val line1: String = "123 Main Street",
         val city: String = "San Francisco",
         val zip: String = "12345",
-        val state: String = "CA",
+        val state: String = "California",
```

Comment:
This is interesting, I think that it should remain as "CA", could be a bug from the work I was doing with state dropdown. I can take a look, I think we can still merge this as is.